tests/e2e/lifecycle/mcg/test_mcg_namespace_lifecyle.py

```diff
@@ -0,0 +1,196 @@
+import json
+import logging
+import uuid
+
+import pytest
+import botocore.exceptions as boto3exception
+
+from ocs_ci.framework.pytest_customization.marks import skipif_aws_creds_are_missing
+from ocs_ci.framework.testlib import E2ETest, tier2, skipif_ocs_version
+from ocs_ci.ocs.bucket_utils import (
+    sync_object_directory, put_bucket_policy,
+    get_bucket_policy, s3_put_object,
+    s3_get_object, s3_delete_object,
+    namespace_bucket_update, rm_object_recursive
+)
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+from ocs_ci.ocs.resources.bucket_policy import NoobaaAccount, gen_bucket_policy, HttpResponseParser
+
+logger = logging.getLogger(__name__)
+
+MCG_NS_RESULT_DIR = '/result'
+MCG_NS_ORIGINAL_DIR = '/original'
+
+
+def setup_base_objects(awscli_pod, amount=2):
+    """
+    Setup function
```

Comment:
It would be better to add some description of what the function does like:
```
Prepare two directories and populate one of them with objects.
```

Reply:
Ack